test.sh

```diff
@@ -28,6 +28,4 @@ fi
 cd ../../
 
 
-print $EXIT_CODE
```

Comment:
nice ;)

Reply:
A PR is not really a PR without debug code left in it. :D